module_analysis/tests/test_module.py

```diff
@@ -28,17 +27,49 @@ def test_installed_modules(self):
                 or module.xml_code_qty > 0
                 or module.js_code_qty > 0,
                 _(
-                    "module %s doesn't have code analysed defined, whereas it is"
-                    " installed."
-                )
-                % (module.name),
+                    "module {} doesn't have code analysed defined, "
```

Comment:
"module %(module_name)s doesn't have code analysed defined, "

is easier to translate in weblate tools.